packages/client/src/routes.ts

```diff
@@ -4,13 +4,20 @@ import { RouteProps } from "react-router-dom";
 import slugify from "slugify";
 
 import { topics } from "./config";
-import topicsJson from "./topics.json";
+import apiTopics from "./topics.json";
+import { TopicType } from "./types";
 
 type RedirectRule = {
   from: string;
   to: string;
 };
 
+type RoutePropExtended = RouteProps & { name: string };
```

Comment:
I find myself doing this stuff too! Interesting pattern.

Reply:
Yeah, it's quite useful to combine props like this